qt/draw_widget.cpp

```diff
@@ -233,13 +237,30 @@ void DrawWidget::mouseMoveEvent(QMouseEvent * e)
   if (IsLeftButton(e) && !IsAltModifier(e))
     m_framework.TouchEvent(GetTouchEvent(e, df::TouchEvent::TOUCH_MOVE));
 
-  if ((m_selectionMode || m_cityBoundariesSelectionMode || m_cityRoadsSelectionMode) &&
+  if ((m_currentSelectionMode) &&
       m_rubberBand != nullptr && m_rubberBand->isVisible())
   {
     m_rubberBand->setGeometry(QRect(m_rubberBandOrigin, e->pos()).normalized());
   }
 }
 
+void DrawWidget::VisualizeMwmsBordersInRect(m2::RectD const & rect, bool withPoints)
+{
+  static std::string const kPathToBorders =
+    base::JoinPath(GetPlatform().ResourcesDir(), "borders");
```

Comment:
Can we have a mode to visualize the borders from packed_polygons.bin?

Reply:
![image](https://user-images.githubusercontent.com/17534533/59373951-6783be80-8d53-11e9-8cd0-1fb7122125c8.png)

Reply:
![image](https://user-images.githubusercontent.com/17534533/59374010-8bdf9b00-8d53-11e9-8413-e90d4afed4d5.png)
Moscow and Moscow Oblast_West with borders from packed_polygon.bin

Reply:
Please also add an option to show the bounding box.

Reply:
![image](https://user-images.githubusercontent.com/17534533/59553630-f6851680-8f9f-11e9-8e54-de1f301fc7c6.png)